egs/wsj/s5/local/chain/e2e/run_tdnn_flatstart.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+# Copyright    2017  Hossein Hadian
+
+# This script performs chain training in a flat-start manner
+# and without building or using any context-dependency tree.
+# It does not use ivecors or other forms of speaker adaptation
+# except simple mean and variance normalization.
+# It is called from run_e2e_phone.sh
+
+# Note: this script is configured as phone-based, if you want
+# to run it in character mode, you'll need to change _nosp
+# to _char everywhere and also copy char_lm.fst instead
+# of phone_lm.fst (in stage 1 below)
+
+# local/chain/compare_wer.sh exp/chain/e2e_tdnn_1a
+# System                      e2e_tdnn_1a
+# WER dev93 (tgpr)                9.41
+# WER dev93 (tg)                  8.88
+# WER dev93 (big-dict,tgpr)       7.12
+# WER dev93 (big-dict,fg)         6.33
+# WER eval92 (tgpr)               5.71
+# WER eval92 (tg)                 5.32
+# WER eval92 (big-dict,tgpr)      3.65
+# WER eval92 (big-dict,fg)        2.85
+# Final train prob        -0.7611
+# Final valid prob        -0.7813
+# Final train prob (xent)
+# Final valid prob (xent)
+# Num-params                 4827732
+
+# steps/info/chain_dir_info.pl exp/chain/e2e_tdnn_1a
+# exp/chain/e2e_tdnn_1a: num-iters=138 nj=2..5 num-params=4.8M dim=40->84 combine=-0.810->-0.809 logprob:train/valid[91,137,final]=(-0.769,-0.761,-0.761/-0.783,-0.782,-0.781)
+
+set -e
+
+# configs for 'chain'
+stage=0
+train_stage=-10
+get_egs_stage=-10
+affix=1a
+decode_iter=
+
+# training options
+num_epochs=4.5
+initial_effective_lrate=0.001
```

Comment:
I think a lot of options are presented here. Some of them can be fixed and removed if they are not too important for tuning.